internal/binarylog/binarylog.go

```diff
@@ -161,11 +161,11 @@ func (l *logger) setBlacklist(method string) error {
 	return nil
 }
 
-// getMethodLogger returns the methodLogger for the given methodName.
+// getMethodLogger returns the BinMethodLogger for the given methodName.
```

Comment:
This still returns `MethodLogger`.  Why the change to `BinMethodLogger` in the comment?

Reply:
Well it sort of returns a BinMethodLogger, which implements MethodLogger (the actual return type). Switched back. Also, the original comment was referring to the specific implementation (methodLogger vs. MethodLogger).

Reply:
If it always returns a `BinMethodLogger` then should we change the signature of the function?  Or it's that way to satisfy an interface and we can't change the interface?

Reply:
Discussed offline; need this method signature to satisfy interface. I did think some comments were incorrect though (in master) and switched them.